src/main/java/org/snomed/cdsservice/service/model/ManyToOneMapEntry.java

```diff
@@ -0,0 +1,36 @@
+package org.snomed.cdsservice.service.model;
+
+import java.util.Set;
+
+public class ManyToOneMapEntry {
```

Comment:
@kaicode is it okay if we replace this record ?
`public ManyToOneMapEntry (Set<String> sourceCodes, String targetCode, int mapPriority, String label) {}`

Reply:
Sure, no problem.